artview/components/select_region.py

```diff
@@ -13,7 +14,7 @@
 
 from ..core import (Variable, Component, common, VariableChoose,
                     componentsList, QtGui, QtCore)
-from ..core.points import Points, write_points_csv, read_points_csv
+from ..core.points import Points
 
```

Comment:
Is there still a way to save the points to a document? Or are you thinking that having a list of polygons will allow the same functionality?